socs/agents/ucsc_radiometer/api/gunicorn.conf.py

```diff
@@ -0,0 +1 @@
+bind = "0.0.0.0:5002"
```

Comment:
This conf has port 5002, but the docs all have port 5000. I think this config might need to be updated.

5000 is the default Flask port, so might be crowded though...